DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs

```diff
@@ -61,14 +61,33 @@ public override async Task OnEvent(IDictionary<string, object> message)
                 { 
                     var status = message["Status"]?.ToString() ?? "";
 
-                    await _viewModel.SetStatus(status);
+                    _viewModel.SetStatus(status);
                     await _viewModel.AppendOutputLine("Status: " + status);
                 }
             }
             else
                 await _viewModel.AppendOutputLine(" Event: " + eventName);
         }
 
+        public override Task OnCommand(string command, dynamic parms)
+        {
+            if(command == "Measurement System")
```

Comment:
is there a glossary of commands (to understand which all commands are there, and what are capabilities of each of those commands) in the app sample?

Reply:
These commands are defined in the manifest